revise pull request template

chicco785 opened the pull request:
<!-- Thank you for submitting a pull request to our repo. -->

- [ ] You've included unit or integration tests for your change, where applicable.
- [ ] You've included inline docs for your change, where applicable.
- [x] There's an open issue for the PR that you are making
    If you'd like to propose a new feature or change,
    please open an issue to discuss the change or find an existing issue.

<!-- Once all that is done, you're ready to go. Open the PR with the content below. -->

## Summary

* include comments in `<!-- -->`
* remove duplicate title
* add note on importance of PR title for release notes

## Description

Fixes #1

kwitekrac commented:
The draft of my idea:
<!-- Thank you for submitting a pull request to our repo. -->

## Description

<!-- Please provide a brief description of the changes in this pull request. -->

## Changes Made

<!-- Please list the changes that you made in this pull request. -->

## Related Issues

Please list any related issues or pull requests.

## Checklist

<!-- Please check off the following items by putting an "x" in the box: -->

- [ ] I have tested these changes locally.
- [ ] I have tested these changes on a dedicated VM or a customer VM [name of the VM]
- [ ] I have added appropriate documentation or updated existing documentation.
- [ ] I have added appropriate tests or updated existing tests.

chicco785 commented:
> The draft of my idea:
> 
> ## Description
> ## Changes Made
> ## Related Issues
> Please list any related issues or pull requests.
> 
> ## Checklist
> * [ ]  I have tested these changes locally.
> * [ ]  I have tested these changes on a dedicated VM or a customer VM [name of the VM]
> * [ ]  I have added appropriate documentation or updated existing documentation.
> * [ ]  I have added appropriate tests or updated existing tests.

see the new version and let me know!
